examples/energy_mng/ui/theme.slint

```diff
@@ -58,5 +119,25 @@ export global Theme {
     in property <Durations> durations: {
         fast: 125ms,
         medium: 200ms,
+        slow: 1200ms,
+    };
+
+    in property <Typo> typo: {
```

Comment:
I suggest to rename this to Typography or TypographySettings or so :)

Reply:
Why such a long name? I like it to be short ;-).

Reply:
True, less characters means lower chances to introduce a ... typo ;-)

Reply:
😄